tests/sphere_lithostatic_pressure_2d.prm

```diff
@@ -0,0 +1,152 @@
+# this is a version of the stokes' law input file
+# but with less mesh refinement
+
+
+
+############### Global parameters
+# We use a 2d setup. Since we are only interested
+# in a steady state solution, we set the end time
+# equal to the start time to force a single time
+# step before the program terminates.
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Pressure normalization                 = surface
+
+
+
+############### Parameters describing the model
+# The setup is a 2d box with edge length 2890000 in which
+# all sides except the left have free slip boundary conditions. Because
+# the temperature plays no role in this model we need not
+# bother to describe temperature boundary conditions or
+# the material parameters that pertain to the temperature.
+
+
+subsection Geometry model
+  set Model name = box
```

Comment:
Your test files are called `sphere_lithostatic_pressure_...`, but they use a box geometry. That's a bit confusing. 
Also,  it would be nice to also have a test with one of the spherical geometries because there is all that extra logic that is a bit different for the spherical geometries.